### ISSUE TITLE
Pipe characters should be encoded in returned Location headers

### ISSUE THREAD

**Issue by erdi** [jirutka/maven-badges#3]
Location headers for responses from `https://maven-badges.herokuapp.com/maven-central/:group/:id` return unescaped pipes:
```
curl -i https://maven-badges.herokuapp.com/maven-central/cz.jirutka.rsql/rsql-parser/

HTTP/1.1 302 Moved Temporarily
Date: Mon, 14 Sep 2015 22:01:37 GMT
Connection: keep-alive
Content-Type: text/html;charset=utf-8
Cache-Control: public, max-age=3600
Location: http://search.maven.org/#artifactdetails|cz.jirutka.rsql|rsql-parser|2.0.0-jdk6|
Content-Length: 0
```

While modern browsers deal with such header gracefully, a pipe (`|`) is not an unreserved character as per section 2.3 of RFC3986 and should be encoded as `%7C` otherwise it causes problems when passed to `java.net.URI#create(java.lang.String)`.

**Comment by jirutka** [comment]
I’m curious, why do you use it from Java?

**Comment by erdi** [comment]
I parse the redirection url returned by your service from Java (actually it's Groovy not Java) because I have a crawler that tests my site deployed to gebish.org which looks for broken links on the site and I had to add special handling for the url returned by your service because it was not RFC3986 compliant. 

Does that answer your question?

**Comment by jirutka** [comment]
Yeah, thanks! I’ve already fixed it in a9ed6c0892db739088b9b974c2823674ff701319.

**Comment by erdi** [comment]
I saw it. :)
